npm install blue-button (missing dependencies)

after fresh install, require("blue-button") complains about missing dependencies for chai, xmldom and execSync in test-lib.js

module.js:340
    throw err;
          ^
Error: Cannot find module 'execSync'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/dkachaev/repos/sandbox/bb-tests/node_modules/blue-button/test/test-lib.js:32:11)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)

<!---
@huboard:{"order":33.5,"milestone_order":72,"custom_state":""}
-->

--- a/.gitignore
+++ b/.gitignore
@@ -19,3 +19,7 @@ ccda-explorer/dump_gen_xml/
 ccda-explorer/sample_ccdas/
 
 test/fixtures/files/generated/
+
+test/fixtures/files/generated/CCD_1_gen.xml
+
+test/fixtures/files/generated/CCD_1_gen.xml

--- a/index.js
+++ b/index.js
@@ -21,7 +21,7 @@ exports.parse = parser.parse;
 exports.generateCCDA = require("./lib/generator/ccda/generator.js").genWholeCCDA;
 
 // testing for ccda generation
-exports.testCCDA = require("./test/test-lib.js").testXML;
+//exports.testCCDA = require("./test/test-lib.js").testXML;
 
 //remove old validator.js from /lib
 exports.validator = require("./lib/validator/validator.js");

